src/ImageSharp/Formats/Gif/GifConstants.cs

```diff
@@ -21,6 +21,11 @@ internal static class GifConstants
         /// </summary>
         public const string FileVersion = "89a";
 
+        /// <summary>
+        /// The ASCII encoded bytes used to identify the GIF file.
+        /// </summary>
+        internal static readonly byte[] MagicNumber = { 71, 73, 70, 56, 57, 97 }; // GIF89a
```

Comment:
👍 A good, simple win. We could use `Encoding.UTF8.GetBytes(FileType + FileVersion)` though to make it a little less "Magic"